internal/common/objects/BaseBiker.go

```diff
@@ -262,6 +270,15 @@ func (bb *BaseBiker) DecideJoining(pendingAgents []uuid.UUID) map[uuid.UUID]bool
 	return decision
 }
 
+// base biker defaults to democracy
```

Comment:
Doesn't this default to Leadership rather than democracy as both democracy and dictatorship are 0 while leadership is 1?

Reply:
yes lol we changed it to check and forgot to change it back... will fix